src/routing/http-routing-v1.md

```diff
@@ -106,17 +106,27 @@ Each object in the `Providers` list is a record conforming to a schema, usually
 
 ```json
 {
-    "Providers": [
-        {
-            "Protocol": "<protocol_name>",
-            "Schema": "bitswap",
-            ...
-        }
-    ]
+  "Providers": [
+    {
+      "Schema": "announcement",
+      ...
+    }
+  ]
 }
 ```
 
-Each object in the `Providers` list is a *write provider record*.
+
+Each object in the `Providers` list is a *write provider record* entry.
+
+Server SHOULD accept  representing writes is [Announcement Schema](#announcement-schema).
+
+:::warn
+
+TODO: is below a sensible limit?
+
+There SHOULD be no more than 100 `Providers` per request.
```

Comment:
this distinction implies we won't want to use this same protocol to sync overall routing tables between nodes - if there are more than 100 known providers for data, would we expect a different protocol to be used?

Reply:
We could:
- Raise limit on the server
- Ask client to make multiple PUTs
  - or use streaming PUT with `application/x-ndjson`  (see https://github.com/ipfs/specs/pull/378#discussion_r1413292096)